vta/python/vta/testing/simulator.py

```diff
@@ -74,5 +74,14 @@ def tsim_init(hw_lib):
     m = tvm.module.load(lib, "vta-tsim")
     f(m)
 
+def tsim_cycles():
+    """Clear profiler statistics
+
+    Returns
+    -------
+    stats : dict
+        Current profiler statistics
```

Comment:
It appears that we are not returning a dictionary here; perhaps we should to accommodate for more profile information (and use the "cycles" as a key for the cycle count)

Reply:
We are not because I am just returning a cycle element, I was thinking that once we think on a second counter, one could added.